debian/patches/01_makefile_fixes.patch

```diff
@@ -16,7 +16,7 @@ This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
 -CC=gcc
 -CFLAGS=-O2 -DNXT	# on some systems add: -I/usr/include
 +CC?=gcc
-+CFLAGS?=-O2 -DNXT	# on some systems add: -I/usr/include
++CFLAGS+=-O2 -DNXT	# on some systems add: -I/usr/include
```

Comment:
This is good - it makes it much easier to experiment with different values of flags, and/or allow debhelper/dpkg to add additional flags.  Same for `gcc` - sometimes it's nice to be able to force a specific version of the compiler in an environment where more than one is installed...